tests/functional/test.cli.run.js

```diff
@@ -47,3 +47,44 @@ describe('web-ext run', () => {
          });
        }));
 });
+
+describe('web-ext run', () => {
```

Comment:
No, that test does other things:
* It invokes the `cmd.run()` function directly so it doesn't cover any command line option parsing
* It is making sure that the `firefoxApp` object (which represents the Firefox application process) is getting called correctly which is something separate